Tasks/Common/webdeployment-common/ltxdomutility.ts

```diff
@@ -2,89 +2,96 @@ var ltx = require("ltx");
 var varUtility = require("./variableutility.js");
 var Q = require('q');
 
-var xmlDomLookUpTable = {};
-var headerContent;
+export class LtxDomUtility  {
 
-export function initializeDOM(xmlContent) {
-    xmlDomLookUpTable = {};
-    headerContent = null;
-    var xmlDom = ltx.parse(xmlContent);
-    readHeader(xmlContent);
-    buildLookUpTable(xmlDom);
-    return xmlDom;
-}
+    private xmlDomLookUpTable = {};
```

Comment:
Can we assign type to each variable?

Reply:
To keep it simple skipping it as we have custom types belonging to different modules.